main.go

```diff
@@ -143,7 +148,7 @@ func main() {
 
 	//Index page endpoints, no Authorization required
 	homeController := controllers.NewHomeController(svc, indexHtml)
-	e.GET("/", homeController.Home)
+	e.GET("/", homeController.Home, createCacheClient().Middleware())
```

Comment:
tested by checking that only first time home endpoint is hit and then it is cached, please check parameters, i took default